png-resizer/app/controllers/Resizer.scala

```diff
@@ -86,7 +92,7 @@ object Resizer extends Controller with Logging {
               resized <- Time(eitherTRight(Im4Java.resizeBufferedImage(width)(bytesPreResize)), "resize image", PngResizerMetrics.resizeTime)
               quanted <- Time(eitherTRight(PngQuant(resized, quality)), "quantize image", PngResizerMetrics.quantizeTime)
               result = Ok(quanted).as("image/png").withHeaders(cacheHeaders: _*)
-            } yield (result)).run.map(_.fold(identity,identity))
+            } yield result).run.map(_.fold(identity,identity))
```

Comment:
I don't understand the use of `Either` if you are just going to `fold` straight out of it with `identity` at the end.